.ci/azure-pipelines-api-client.yml

```diff
@@ -0,0 +1,62 @@
+﻿parameters:
+  - name: LinuxImage
+    type: string
+    default: "ubuntu-latest"
+  - name: GeneratorVersion
+    type: string
+    default: "4.3.1"
+
+jobs:
+- job: GenerateApiClients
+  displayName: 'Generate Api Clients'
+
+  pool:
+    vmImage: "${{ parameters.LinuxImage }}"
+
+  steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: 'Download OpenAPI Spec Artifact'
+      inputs:
+        source: 'current'
+        artifact: "OpenAPI Spec"
+        path: "$(System.ArtifactsDirectory)/openapispec"
+        runVersion: "latest"
+        dependsOn: Test
+
+    - task: CmdLine@2
+    displayName: 'Download OpenApi Generator'
```

Comment:
This indentation, is that correct? Use the Azure Pipelines vs code plugin to validate the yaml. I think it needs an extra two spaces.